sdk/compute/Microsoft.Azure.Management.Compute/src/Generated/DiskRestorePointOperations.cs

```diff
@@ -474,6 +474,506 @@ public async Task<AzureOperationResponse<IPage<DiskRestorePoint>>> ListByRestore
             return _result;
         }
 
+        /// <summary>
```

Comment:
1)	You must add or edit at least 1 test to show the feature is functioning as intended. 
2)	You should update the version number of the .Net SDK package you are updating. 
Be sure to also update the Release Notes with a description of what is in the release (your feature). 
https://github.com/Azure/azure-sdk-for-net/pull/20093/files#diff-9b3521ba337eac7e99f3032128aa917a24b94a21cd0fd16fee05cff73e496d35R15